runners/core-java/src/main/java/org/apache/beam/runners/core/SplittableParDo.java

```diff
@@ -391,19 +392,19 @@ public PipelineOptions getPipelineOptions() {
       };
     }
 
-    /** Creates an {@link DoFn.ExtraContextFactory} that provides just the given tracker. */
-    private DoFn.ExtraContextFactory<InputT, OutputT> wrapTracker(
+    /** Creates an {@link Arguments} that provides just the given tracker. */
+    private Arguments<InputT, OutputT> wrapTracker(
         TrackerT tracker, DoFn<InputT, OutputT>.ProcessContext processContext) {
-      return new ExtraContextFactoryForTracker<>(tracker, processContext);
+      return new ArgumentsForTracker<>(tracker, processContext);
     }
 
-    private static class ExtraContextFactoryForTracker<
+    private static class ArgumentsForTracker<
```

Comment:
Is this really `ArgumentsForTracker`? Based on the description it seems more like arguments to either (1) a specific method or set of methods, or (2) arguments only include the tracker and such.

Why is this necessary -- couldn't we just provide the underlying arguments? The validation should make sure the user won't get any of the unsupported argument types already, so this seems like extra overhead...